test/unit/migrations/multiple_ldap_hostnames_migration_test.rb

```diff
@@ -0,0 +1,104 @@
+# encoding: utf-8
```

Comment:
never rely on return value of up and down. just because ruby returns the last statement doesn't mean you should use it for testing. the return value here is never used in the application. so please check if the respective setting has been updated correctly by loading and checking it's value.